files/scripts/swss.sh

```diff
@@ -131,7 +131,13 @@ start() {
 
 wait() {
     start_peer_and_dependent_services
-    /usr/bin/${SERVICE}.sh wait
+
+    # Allow peer service some time to start up
+    sleep 5
```

Comment:
why is this needed before?

Reply:
This is due to the addition of the `docker-wait-any` script. I want to ensure that the syncd container is up and running before starting `docker-wait-any`, because if it isn't running when the script is kicked off, the script will exit immediately.

Reply:
it is better to have a loop per second, and check if syncd started, if yes then call docker-wait-any.

Reply:
What if for some reason the syncd container fails to start? Should we set a max wait time?

Reply:
yes, for example, 60 seconds, which is much safer.

Reply:
Done in commit 3a9028ebe5cd7f4fc6334e7a3194b378762f7f6a